src/resources/Enums.ts

```diff
@@ -844,6 +844,42 @@ export enum LicenseSection {
     searchapi = 'searchapi',
 }
 
+export enum SearchHubRawMetrics {
+    normalQueries = 'normalQueries',
+    recommendationQueries = 'recommendationQueries',
+    commerceProductListingQueries = 'commerceProductListingQueries',
+    users = 'users',
+    staticQueries = 'staticQueries',
+}
+
+export enum SearchHubLicenseMetrics {
+    normalQueries = 'normalQueries',
+    recommendationQueries = 'recommendationQueries',
+    commerceProductListingQueries = 'commerceProductListingQueries',
+    agents = 'agents',
+    staticQueries = 'staticQueries',
+    intranetUsers = 'intranetUsers',
+    partnetUsers = 'partnerUsers',
+    timesExceededQps = 'timesExceededQps',
+    platformIntranetUsers = 'platformIntranetUsers',
+    platformAgents = 'platformAgents',
+    platformPortalUsers = 'platformPortalUsers',
+    salesforceServiceCloudAgents = 'salesforceServiceCloudAgents',
+    salesforceSalesCloudUsers = 'salesforceSalesCloudUsers',
+    salesforcePlatformUsers = 'salesforcePlatformUsers',
+    salesforceCommunityCloudUsers = 'salesforceCommunityCloudUsers',
+    snowCSMAgents = 'snowCSMAgents',
+    snowFulfillers = 'snowFulfillers',
+    snowITSMUsers = 'snowITSMUsers',
+    snowHRSDUsers = 'snowHRSDUsers',
+    dynamicsCustomerEngagementUsers = 'dynamicsCustomerEngagementUsers',
+}
+
+export enum SearchHubLicenseType {
+    Assignment = 'assignment',
+    Global = 'global',
```

Comment:
be consistent in enum naming 
```suggestion
    assignment = 'assignment',
    global = 'global',
```